io_scene_godot/converters/animation.py

```diff
@@ -379,6 +379,16 @@ def bake_constraint_to_action(blender_object, base_action,
             do_visual_keying=True,
         )
 
+    if in_place:
+        return action_bake_into
+
+    if base_action is not None:
+        # blender would automatically add a '.001' for a name collision
+        # hacky solution by adding a space to make name different,
+        # space is ignored in godot animation name
+        baked_action.name = ' ' + base_action.name
```

Comment:
Doesn't this mean that you'll get a bunch of identicly named actions in godot's animation window. I think that would be quite a pain to try and use (particuarly if you're trying to trigger those actions from gscript). In blender, actions have names, so I wonder if we can actually use the action name.

Reply:
No, exported actions is the same number of actions you have in blender.    The baked action is always new created because I found if bake into the original action, blender would not set the original action back when addon finish exporting,  so the baked action is created as a temporal one and would be removed after it get evaluated and exported